### DISCUSSION TITLE
Wikijs logs location

### DISCUSSION THREAD

**Discussion by deeepakkumarthakur16** [requarks/wiki#5855] [Help / Questions] [answered]
@NGPixel Can you tell me the logs location for wikijs as I am not able to find any logs location in wikijs directory or in config file.

**Comment by NGPixel** [comment, marked as answer]
There's no logs location. You need to look at the output. If using docker, you would use `docker logs wiki`. For native, almost all process / service managers have an option to save the output to a log file.

**Reply by deeepakkumarthakur16** [reply]
thanks. I am not using docker but wiki should have a place where all logs should be stored for audit or compliance purpose.

**Comment by dr0924** [comment]
Docker or not docker, there should be logs written somewhere with basic info level and for troubleshooting purposes, there should be increased level of debug. That is the basic standard for any software.  I see there are some logs written in reverse proxy server logs location. ( ex: /var/log/nginx/) but that may not have proper information as it only what it can capture during reverse proxy.

**Reply by NGPixel** [reply]
As explained above, logs are written to standard output, **which can be saved to disk at the service level** if that's what you require. Log level can be set using the `logLevel` setting in your config.yml file.
